lib/src/ui/barcode_scanner_widget.dart

```diff
@@ -137,6 +158,33 @@ class _BarcodeScannerWidgetState extends State<BarcodeScannerWidget> {
     }
   }
 
+  Future<bool> _onSupportsSwitchingCameraListener() async {
+    for (final value in _configuredScanners) {
+      if (await value?.controller.supportsSwitchingCamera ?? false) return true;
+    }
+    return false;
+  }
+
+  bool _onTorchStateListener() => _configuredScanners
+      .any((element) => element?.controller.torchState ?? false);
+
+  Future<void> _onToggleCameraListener() async {
+    for (final value in _configuredScanners) {
+      if (await value?.controller.supportsSwitchingCamera ?? false) {
+        value?.controller.toggleCamera();
+        return;
```

Comment:
why not toggle everything if possible?

Reply:
Because if the toggle is implemented in multiple controllers correctly, it will maybe toggle an even number of times and makes it that it didn't toggle

Reply:
But let's say you have 4 scanners
- scanner 1 (can't switch)
- scanner 2 (can switch)
- scanner 3 (can't switch)
- scanner 4 (can switch)

when you call this method. You will check if the first one can switch. no -> next one -> yes -> return. 

So now only scanner 2 switched. and scanner 4 did not switch? That is not correct right? Or am I seeing something else wrong?

Reply:
Aaaaah yes I see 🤦